RockWeb/Blocks/Security/AccountEntry.ascx.cs

```diff
@@ -259,7 +259,10 @@ protected void btnDuplicatesNext_Click( object sender, EventArgs e )
             if ( personId > 0 )
             {
                 var userLoginService = new Rock.Model.UserLoginService( new RockContext() );
-                var userLogins = userLoginService.GetByPersonId( personId ).ToList();
+				var databaseEntityTypeId = EntityTypeCache.Read( Rock.SystemGuid.EntityType.AUTHENTICATION_DATABASE.AsGuid() ).Id;
+                var userLogins = userLoginService.GetByPersonId( personId )
+				.Where( ul => ul.EntityTypeId == databaseEntityTypeId )
+				.ToList();
```

Comment:
@trydyingtolive Instead of hard-coding the Database authentication type, can you instead check to see if they have any logins that don't RequiresRemoteAuthentication. With this change, if person had an ActiveDirectory login, it would not send them the login id.

Here's an example of code that looks for login types that don't require the remote authentication: https://github.com/SparkDevNetwork/Rock/blob/develop/RockWeb/Blocks/Security/ForgotUserName.ascx.cs#L104